qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/customnavview/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -236,6 +257,48 @@ class MapboxNavigationViewCustomizedActivity : DrawerActivity() {
         }
     }
 
+    private fun toggleCustomInfoPanelContent(enabled: Boolean) {
+        binding.navigationView.customizeViewBinders {
+            if (enabled) {
+                infoPanelContentBinder = UIBinder { viewGroup ->
+                    supportFragmentManager.beginTransaction()
+                        .replace(viewGroup.id, CustomInfoPanelDetailsFragment())
+                        .commitAllowingStateLoss()
+                    UIComponent()
+                }
+            } else {
+                infoPanelContentBinder = UIBinder.USE_DEFAULT
+            }
+        }
+    }
+
+    private fun toggleCustomInfoPanelLayout(enabled: Boolean) {
+        binding.navigationView.customizeViewBinders {
+            infoPanelBinder =
+                if (enabled) CustomInfoPanelBinder()
+                else InfoPanelBinder.defaultBinder()
+        }
+        binding.navigationView.customizeViewStyles {
+            infoPanelPeekHeight =
+                if (enabled) defaultInfoPanelPeekHeight(applicationContext) + 20.dp
+                else defaultInfoPanelPeekHeight(applicationContext)
+        }
+    }
+
+    private fun toggleCustomInfoPanelStyles(enabled: Boolean) {
+        binding.navigationView.customizeViewStyles {
+            if (enabled) {
+                infoPanelBackground = R.drawable.bg_custom_info_panel2
+                infoPanelMarginStart = 5.dp
+                infoPanelMarginEnd = 5.dp
```

Comment:
```suggestion
                infoPanelMarginStart = 10.dp
                infoPanelMarginEnd = 10.dp
```
So that it's much more visible.